src/docfx/config/SecretConfig.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    /// <summary>
+    /// Add a comment to say all secrets <b>MUST</b> go here otherwise they are print out to logs
```

Comment:
```suggestion
    /// All secrets <b>MUST</b> go here otherwise they are print out to logs
```
Based on my understanding of the comment below :)